latex/interni/doc_interna/norme_di_progetto/res/sections/01_content.tex

```diff
@@ -181,26 +206,27 @@ \subsubsection{Commits}
 
 \subsubsection{Pull Requests}
 Per effettuare un merge su un branch protetto si deve aprire da GitHub\textsubscript{g} una Pull Request.
-La Pull Request permette di verificare il lavoro svolto prima di integrarlo con il branch desiderato.
-I verificatori in carica hanno il compito di trovare eventuali errori o mancanze e fornire un feedback riguardante il contenuto direttamente su GitHub\textsubscript{g} richiedendo
+La Pull Request permette di verificare il lavoro svolto prima di integrarlo con il branch desiderato.\\
+Alla creazione di una Pull Request bisogna associare:
+\begin{itemize}
+\item i verificatori in carica hanno il compito di trovare eventuali errori o mancanze e fornire un feedback riguardante il contenuto direttamente su GitHub\textsubscript{g} richiedendo
 una review con un review comment sulla parte specifica da revisionare o con un commento generico.\\
 Non sarà possibile effettuare il merge finchè tutti i commenti di revisione non saranno stati risolti.
 Il merge potrà avvenire dopo aver risolto tutti i commenti di revisione ed approvato da almeno due verificatori.
```

Comment:
aggiungere un che  "i verificatori in carica che hanno il compito"... troppe ripetizioni da riga 213 a 215.. raggruppare il tutto nella frase "Non sarà possibile effettuare il merge finchè tutti i commenti di revisione non saranno stati risolti e la Pull Request approvata da due verificatori"

Reply:
ok